### PR TITLE
Remove crd manifest which is now in cluster-config-operator

### PR DESCRIPTION
**Pull request by sttts**
Depends on https://github.com/openshift/cluster-config-operator/pull/23.

### CODE REVIEW

File: manifests/0000_07_cluster-network-operator_01_crd.yaml [hunk 1]
```diff
@@ -1,115 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: networkconfigs.networkoperator.openshift.io
```
Comment (danwinship) [inline manifests/0000_07_cluster-network-operator_01_crd.yaml]: This file installs both `networks.config.openshift.io` and `networkconfigs.networkoperator.openshift.io` (the operator-specific config). cluster-config-operator only installs the former, so you can't get rid of this file completely.

Comment (sttts) [reply manifests/0000_07_cluster-network-operator_01_crd.yaml]: fixed

**Comment by danwinship** [comment]
sorry, forgot to re-review this, and now it seems soltysh submitted a duplicate PR (#124) which is already lgtm'ed

**Pull request closed without merging**